src/main/resources/hudson/plugins/jira/JiraSite/config.jelly

```diff
@@ -6,8 +6,11 @@
   <f:entry title="Link URL" field="alternativeUrl" description="${%site.alternativeUrl}">
     <f:textbox />
   </f:entry>
-  <f:entry>
+  <f:invisibleEntry>
     <f:checkbox title="${%Use HTTP authentication instead of normal login}" field="useHTTPAuth" />
+  </f:invisibleEntry>
+  <f:entry>
+    <f:checkbox title="${%Use Bearer authentication instead of Basic authentication}" field="useBearerAuth" description="${%Note: Bearer authentication is only supported in Jira Server, for Jira Cloud leave this unchecked}" />
```

Comment:
Almost :-) This should be: `${%site.useBearerAuth}` and then [config.properties](https://github.com/jenkinsci/jira-plugin/blob/master/src/main/resources/hudson/plugins/jira/JiraSite/config.properties):
```
site.useBearerAuth=....
```

Reply:
🤦 Sorry